octobot_evaluators/api/evaluators.py

```diff
@@ -58,18 +58,15 @@ def get_evaluator_classes_from_type(evaluator_type, tentacles_setup_config, acti
     return tentacles_management.get_all_classes_from_parent(evaluator.EvaluatorClassTypes[evaluator_type])
 
 
-async def initialize_evaluators(config, tentacles_setup_config) -> str:
+async def initialize_evaluators(config, tentacles_setup_config) -> None:
     """
     :param config: bot config
     :param tentacles_setup_config: tentacles configuration
-    :return: initialized matrix id
     """
     _init_time_frames(config, tentacles_setup_config)
     # take evaluators and strategies candles requirements into account if any
     api.init_required_candles_count_from_evaluators_and_strategies(config, tentacles_setup_config)
 
-    return create_matrix()
```

Comment:
It's not used?

Reply:
it is but for the backtester setup, we need to create it before creating evaluators so I removed this call and we now have to call both `create_matrix` and `initialize_evaluators`, which makes more sense in my opinion compared to a `initialize_evaluators` that also creates a new matrix (I updated the code using it in other PRs to have the same behavior as before)

Reply:
👍